sqm-scripts/patches/000-OverTheBox-SQM-run.patch

```diff
@@ -1,162 +1,74 @@
-diff --git a/luci/sqm-cbi.lua b/luci/sqm-cbi.lua
-index 9b8af51..fd5dd53 100644
---- a/luci/sqm-cbi.lua
-+++ b/luci/sqm-cbi.lua
-@@ -13,6 +13,7 @@ You may obtain a copy of the License at
- $Id$
- ]]--
- 
-+local uci = luci.model.uci.cursor()
- local wa = require "luci.tools.webadmin"
- local fs = require "nixio.fs"
- local net = require "luci.model.network".init()
-@@ -55,17 +56,48 @@ function e.write(self, section, value)
- 	end
- 	return Flag.write(self, section, value)
- end
---- TODO: inform the user what we just did...
- 
--n = s:taboption("tab_basic", ListValue, "interface", translate("Interface name"))
-+-- TODO: inform the user what we just did...
-+n = s:taboption("tab_basic", ListValue, "_interface_all", translate("Interface name"))
- -- sm lifted from luci-app-wol, the original implementation failed to show pppoe-ge00 type interface names
- for _, iface in ipairs(ifaces) do
- 	if not (iface == "lo" or iface:match("^ifb.*")) then
--		n:value(iface)
-+		local multipath = uci:get("network", iface, "multipath")
-+		if multipath == "on" or multipath == "backup" or multipath == "master" or multipath == "handover" then
-+			n:value(iface, (uci:get("network", iface, "label") or "") .. " (" .. iface .. ")" )
-+		else
-+			n:value(iface)
-+		end
- 	end
- end
- n.rmempty = false
-+n.write = function(self, section, value)
-+	m:set(section, "interface", value)
-+end
-+n.cfgvalue = function(self, section)
-+	return m:get(section, "interface", value)
-+end
-+n.optional = true
- 
-+-- TODO: inform the user what we just did...
-+n1 = s:taboption("tab_basic", ListValue, "_interface_mptcp", translate("Interface name"))
```

Comment:
Just some temp vars to build the form i can name it "useless_var_n" and "useless_var_n1" if you like